app/payment-link-v2/reference/reference.njk

```diff
@@ -0,0 +1,59 @@
+{% extends "../../views/layout.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "../../views/macros/error-summary.njk" import errorSummary %}
+
+{% block pageTitle %}
+  {{ __p('paymentLinksV2.amount.pleaseEnterYour') }} {{product.reference_label}}  - {{ productName }}
+{% endblock %}
+
+{% block contentBody %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    {{ govukBackLink({
+      text: "Back",
+      href: backLinkHref
+    }) }}
+
+    {{ errorSummary ({
+      errors: errors,
+      hrefs: {
+        'payment-reference': '#payment-reference'
+      }
+    }) }}
+
+    <form action="/pay/{{ productExternalId }}/reference" method="post" name="startPaymentForm" class="push-bottom">
```

Comment:
you can just leave out the `action` attribute, and it will POST to the same URL as the page is already on to simplify things ever so slightly

Reply:
Didn't know that - I will remove the `action` attributes.